packages/steppers/src/styled/StyledStepContent.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { TransitionStatus } from 'react-transition-group/Transition';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'steppers.step_content';
+
+export interface ITransitionStyles {
+  transitionStatus?: TransitionStatus;
+}
+
+export interface IStyledStepContent extends ITransitionStyles {
+  isActive?: boolean;
+}
+
+const transitionStyles = (props: ITransitionStyles & ThemeProps<DefaultTheme>) => {
+  if (props.transitionStatus === 'exiting') {
+    return css`
+      height: auto;
+      max-height: 0 !important; /* stylelint-disable-line */
+    `;
+  }
+
+  if (props.transitionStatus === 'exited') {
+    return css`
+      max-height: 0 !important; /* stylelint-disable-line */
+    `;
+  }
+
+  return undefined;
+};
+
+export const StyledStepContent = styled.div.attrs<IStyledStepContent>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledStepContent>`
+  transition: max-height 0.5s ease-in-out;
```

Comment:
These transition values may need to be changed. I went with what looked natural, however, I don't have a good set of heuristics for determining these values. I looked at `Chrome`'s subnav transition, however, I thought that it transitioned a bit quickly and that a `Stepper` animates a bit differently (the previous step has an exit transition).

@ginnywood @jzempel feel free to provide new transitions value in this thread, its heuristics, and I can make adjustments to the transition 👍.

Reply:
It feels a little sluggish to me right now. I think it could speed up a bit.
The main issue is sequencing: when the new step is transitioning to active, it first opens, _then_ slides up into place. Instead, it should happen simultaneously, sliding up into the area previously occupied by the previous step, while opening at the same time.
Let’s talk about this.